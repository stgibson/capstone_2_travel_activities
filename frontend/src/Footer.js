import React from "react";

const Footer = () => {
  return (
    <>
      <div>
        <small>
          The information on travel activities in this website is provided by the
          Tours and Activities API by Amadeus for Developers located at
          https://developers.amadeus.com/self-service/category/destination-content/api-doc/tours-and-activities/api-reference
        </small>
      </div>
      <div>
        <small>
          Furthermore, this site also uses the Geocoding API located at
          https://developers.google.com/maps/documentation/geocoding/overview
        </small>
      </div>
    </>    
  );
};

export default Footer;