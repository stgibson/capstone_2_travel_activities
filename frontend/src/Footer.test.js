import React from "react";
import { render } from "@testing-library/react";
import Footer from "./Footer";

it("renders without crashing", () => {
  render(<Footer />);
});

it("matches snapshot", () => {
  const { asFragment } = render(<Footer />);

  expect(asFragment()).toMatchSnapshot();
});
