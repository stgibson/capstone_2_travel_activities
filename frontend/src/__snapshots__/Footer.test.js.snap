// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <nav
    class="navbar navbar-expand navbar-light fixed-bottom"
  >
    <small>
      The information on travel activities in this website is provided by the Tours and Activities API by Amadeus for Developers located at https://developers.amadeus.com/self-service/category/destination-content/api-doc/tours-and-activities/api-reference. Furthermore, this site also uses the Geocoding API located at https://developers.google.com/maps/documentation/geocoding/overview
    </small>
  </nav>
</DocumentFragment>
`;
